### PR TITLE
Fendpoint to glue method mapping

### PR DESCRIPTION
**Pull request by mpsonntag**
- move all remaining methods from fendpoint mapping to glue mapping
- remove fendpoint mapping

### CODE REVIEW

**Comment by gicmo** [comment]
Ok for now I guess, but some (quite a bit?) could be migrated to GLUE proper. First thing that comes to my mind is the `.desc` stuff.

File: nix_mx.cc [hunk 1]
```diff
@@ -140,6 +96,8 @@ void mexFunction(int            nlhs,
             .reg("deleteSection", REMOVER(nix::Section, nix::File, deleteSection))
             .reg("openBlock", GETBYSTR(nix::Block, nix::File, getBlock))
             .reg("openSection", GETBYSTR(nix::Section, nix::File, getSection));
+        methods->add("File::createBlock", nixfile::create_block);
+        methods->add("File::createSection", nixfile::create_section);
```
Comment (asobolev) [inline nix_mx.cc]: These two should work via matryoschka, right?

**Comment by gicmo** [comment]
LGTM now

**Comment by mpsonntag** [comment]
completely missed the .desc thing, sorry! Included it now.

**Pull request merged**